src/reference/changes.rst

```diff
@@ -172,6 +191,42 @@ For more information, see the reference for the
 :cylc:conf:`[runtime][<namespace>]completion` configuration.
 
 
+Workflow State Triggers & Commands
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Workflow state xtriggers and command now take Cylc universal IDs instead of
+separate arguments:
+
+For example, you can (and should) now write:
+
+.. code-block:: diff
+
+   # On the command line
+   - cylc workflow-state my-workflow --point 20240101 --task mytask --message "succeeded"
+   + cylc workflow-state my-workflow//20240101/mytask:succeeded
```

Comment:
Technically this should be either:
```suggestion
   - cylc workflow-state my-workflow --point 20240101 --task mytask --status "succeeded"
   + cylc workflow-state my-workflow//20240101/mytask:succeeded
```
or
```suggestion
   - cylc workflow-state my-workflow --point 20240101 --task mytask --message "succeeded"
   + cylc workflow-state my-workflow//20240101/mytask:succeeded --triggers
```

Reply:
succeeded is a message, trigger and status, so I think it's ok.

Reply:
The aim here is to illustrate the change from using messages to using triggers.

The second one mismatches with `cylc workflow-state --help`:

>   --triggers            Task selector should match output triggers rather than
                        status. (Note this is not needed for custom outputs).